ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -349,13 +346,12 @@
         }
```

Comment:
Removing `color: #000;` might affect text readability if the parent element doesn't set a default color. Consider keeping it unless the design intentionally inherits the color.